go-controller/pkg/ovn/zone_interconnect/interconnect.go

```diff
@@ -206,10 +217,11 @@ func (zic *ZoneInterconnectHandler) createLocalZoneNodeResources(node *corev1.No
 
 	_, err = libovsdbops.GetLogicalSwitch(zic.nbClient, ts)
 	if err != nil {
+		transitSwitchTunnelKey := BaseTransitSwitchTunnelKey + networkId
```

Comment:
so this is the reason why we need the networkID, gotcha..I want to admit I do have a concern over all the annotations we are adding nodeID, networkID, just for IC alone we have added a lot of annotations onto the node...considering node object is not something a cluster user can modify I guess this is ok.
As part of the IC docs I am writing I'll make sure to try and include all this info.

Reply:
yeah would be nice to have a comment in the commit explaining why you need a networkID, rather than reviewing almost all the way to the end of the commit to find out why. Maybe some comments around networkID  code itself as well.

Reply:
Actually there is a comment in the commit about it.